src/components/Global/DirectSendQR/index.tsx

```
@@ -22,21 +26,23 @@ export default function DirectSendQr() {
-            router.push(path)
-            return { success: true }
+            redirectUrl = path
         } else if (isAddress(data)) {
             // hardcode PNT because this is only used for pinta wallet for now
-            const link = `${baseUrl}/${data}@polygon/PNT`
-            router.push(link)
-            return { success: true }
+            redirectUrl = `${baseUrl}/${data}@polygon/PNT`
         } else if (validateEnsName(data)) {
             const resolvedAddress = await resolveFromEnsName(data.toLowerCase())
             if (!!resolvedAddress) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)